### DISCUSSION TITLE
How to view mobile/responsive version on TatoVM

### DISCUSSION THREAD

**Discussion by vinkaks** [Tatoeba/tatoeba2#2986] [Q&A] [answered]
How do you view the mobile version in the VM. I tried chrome dev tools, no joy.

**Comment by Yorwba** [comment, marked as answer]
I use Firefox, but I'd be surprised if Chrome were significantly worse at simulating a mobile phone. Most likely you simply didn't set up the necessary conditions for the mobile version to be shown.

- If you're logged in, you need to enable the new design in the settings. Only the search results page will be responsive.
- If you're logged out, the home page will be responsive as well.
- Your viewport should be at most 959 pixels wide to trigger the mobile view. (Modern phones tend to have higher resolutions, but mobile browsers pretend that they don't, so that desktop sites remain somewhat readable on small phone screens.)

If your network setup allows it, you can also test using an actual phone. E.g. I run TatoVM on a Linux desktop with the hostname `desktop` (I was in a creative phase when I named it) and I can simply visit `desktop:8080` with my phone's browser while the two devices are on the same network. (I think I might've had to configure my router to "allow devices to talk to each other" or something of the sort.)
